contracts/src/base/registry/facets/delegation/SpaceDelegationFacet.sol

```diff
@@ -51,6 +54,10 @@ contract SpaceDelegationFacet is ISpaceDelegation, OwnableBase, Facet {
     if (currentOperator != address(0) && currentOperator == operator)
       CustomRevert.revertWith(SpaceDelegation__AlreadyDelegated.selector);
 
+    // check if the operator is valid
+    if (!NodeOperatorStorage.layout().operators.contains(operator))
+      CustomRevert.revertWith(SpaceDelegation__InvalidOperator.selector);
```

Comment:
Should we also check status?

Reply:
can probably check for !Exiting, once slashing is available we should revisit this since it could be that you can only delegate to an operator that has staked